src/core/documentation/decision.ts

```diff
@@ -0,0 +1,63 @@
+import { Element } from "../model/element";
+import { Format } from "./format";
+import { DecisionStatus } from "./decisionStatus";
+
+export interface DecisionDto {
+    id: string;
+    date: string;
+    status: DecisionStatus;
+    title: string;
+    content: string;
+    format: Format;
+    elementId?: string;
+}
+
+export class Decision {
+    id: string;
+    date: Date;
+    status: DecisionStatus;
+    element: Element | null;
+    title: string;
+    format: Format;
+    content: string;
+
+    constructor(
+        element: Element | null,
```

Comment:
The same reason as for the DTO interfaces basically goes for the constructors - the constructors are only to be called within this lib, not from the end user, which is why I kept them mostly parameterless in order not to expose too much detail on how they are constructed from DTOs.

Basically any other entity has an empty or nearly empty constructor, and a `fromDto` function. Like your `Documentation` class. Rule of thumb: if a class has a `toDto` function, it should also have a `fromDto`